src/app/selectors/priceChart.js

```diff
@@ -0,0 +1,22 @@
+import { createSelector } from 'reselect'
+import { createItemSelector, selectItemId, fieldSelector } from 'Utilities/selector'
+
+export const getPriceChartState = ({ priceChart }) => priceChart
+
+export const getAllPriceCharts = createSelector(getPriceChartState, (priceCharts) => {
+  return {
+    ...priceCharts
+  }
+})
+
+export const getPriceChartData = createItemSelector(getAllPriceCharts, selectItemId, (allCharts, symbol) => allCharts[symbol])
```

Comment:
This doesn't actually return the price chart data so it should be called `getPriceChart` and you can add another item selector specifically for getting just the data called `getPriceChartData`. Ideally the selectors you use in a component should only return exactly the data you need to avoid unnecessary renders.

Reply:
ohhhh i see ok gotcha